resources/xcode.rb

```diff
@@ -6,6 +6,18 @@
 property :ios_simulators, Array
 
 action :setup do
+  file 'sentinel to make CLT available' do
+    path '/tmp/.com.apple.dt.CommandLineTools.installondemand.in-progress'
+    not_if { ::File.exist?('/Library/Developer/CommandLineTools/usr/lib/libxcrun.dylib') }
+    notifies :run, 'execute[install Xcode CLT]', :immediately
+  end
+
+  execute 'install Xcode CLT' do
+    command lazy { "softwareupdate --verbose -i '#{CommandLineTools.new}'" }
```

Comment:
A couple legitimate questions:
- Based on the unit test, do we need to use `CommandLineTools.new.product` in the command?
- Do we want to be live-streaming here? If not, is the `verbose` option needed?

Nitpicks. Could we:
- extract the command to a list variable for readability (clearer that we need `CommandLineTools` in quotes)
- use the long version of the `install` option

```ruby
execute 'install Xcode CLT' do
  install_clt_command = ['softwareupdate', '--install', "'#{CommandLineTools.new}'"]
  command lazy { install_clt_command }
  action :nothing
  notifies :delete, 'file[sentinel to make CLT available]', :immediately
end
```

Reply:
Yep.

Reply:
`CommandLineTools` also is handled by the command array properly (something I hadn't tested before)